connectors/sources/postgresql.py

```diff
@@ -436,20 +439,20 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: Row dictionary containing meta-data of the row.
         """
-        async for schema in self.postgresql_client.get_all_schemas():
-            if schema not in SYSTEM_SCHEMA:
-                table_count = 0
-                async for table in self.postgresql_client.get_tables_to_fetch(schema):
-                    self._logger.debug(
-                        f"Found table: {table} in database: {self.database}."
-                    )
-                    table_count += 1
-                    async for row in self.fetch_documents(
-                        table=table,
-                        schema=schema,
-                    ):
-                        yield row, None
-                if table_count < 1:
-                    self._logger.warning(
-                        f"Fetched 0 tables for schema: {schema} and database: {self.database}"
-                    )
+        table_count = 0
+
+        async for table in self.postgresql_client.get_tables_to_fetch(self.schema):
```

Comment:
Probably we should consider passing `schema` (or simpler, `configuration`) to `PostgreSQLClient` initializer.

Reply:
I've removed `get_all_schemas` but I wasn't sure on the impact of removing `all_schemas` from `Queries` as other connectors seem dependent on it, so I've just removed the content of the func from the `PostgreSQLQueries` class.

Reply:
Also passed `schema` on to `PostgreSQLClient` so it can be initialized there.